### PR TITLE
refactor: don't show comments on hover on FD/SD variables.

### PR DESCRIPTION
**Pull request by ap891843**
Signed-off-by: ap891843 <aman.prashant@broadcom.com>

### CODE REVIEW

File: server/src/main/antlr4/org/eclipse/lsp/cobol/core/parser/CICSLexer.g4 [hunk 1]
```diff
@@ -13,6 +13,7 @@
  */
 
 lexer grammar CICSLexer;
+channels{COMMENTS}
```
Comment (ap891843) [inline server/src/main/antlr4/org/eclipse/lsp/cobol/core/parser/CICSLexer.g4:16]: Had to add the channel to each lexer, as it seems the channel doesn't work for combined grammar.

ref - https://github.com/antlr/antlr4/issues/1555
ref - https://github.com/antlr/antlr4/issues/965

**Comment by ap891843** [comment]
fixes - https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/1064

**Review by temanbrcom — APPROVED** [review]
🧛

**Comment by temanbrcom** [comment]
Please, change the prefix of the message to "refactor" because the feature was not released yet.

**Review by grianbrcom — APPROVED** [review]
:comet:

**Pull request merged**